index.html

```diff
@@ -20,14 +20,120 @@
   <section class="section2">
     <h1>I'm Ahmed <br> Glad to see <br> you!</h1>
     <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    <a href="#" class="connect">Let's Connect</a>
+    <p class="connect">Let's Connect</p>
     <ul class="socialmeadia-container">
       <li class="icons"><a href="#"><img src="./svgs/Vector.svg" alt="svg"></a></li>
       <li class="icons"><a href="#"><img src="./svgs/Linkedinicon.svg" alt="svg"></a></li>
       <li class="icons"><a href="#"><img src="./svgs/Vector(4).svg" alt="svg"></a></li>
       <li class="icons"><a href="#"><img src="./svgs/Vector(2).svg" alt="svg"></a></li>
       <li class="icons"><a href="#"><img src="./svgs/Vector(3).svg" alt="svg"></a></li>
     </ul>
+  </section><br>
+  <section class="works">
+    <div class="card1">
+      <img src="works/Snapshoot-Portfolio1.svg" alt="snaphot" class="p-img">
+      <h2>Tonic</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5 class="sec">Back End Dev</h5>
```

Comment:
[Optional] I recommend that you modify the image currently within your header (serving as the page's background design) to be a CSS background image. This is because it's not part of the content and only a design. Read more about [when to use CSS background image](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html).

It seems to cause a horizontal overflow of your page, allowing for horizontal scrolling.

Here's your design:
![image](https://user-images.githubusercontent.com/20554583/185738507-4cb83bb5-8f99-414c-ae40-7a3d602e0c6e.png)

Reply:
thank you